mava/systems/tf/madqn/system.py

```diff
@@ -302,7 +309,7 @@ def __init__(  # noqa
         # Check that all agent_net_keys are in trainer_networks
         assert unique_net_keys == unique_trainer_net_keys
         # Setup specs for each network
-        self._net_spec_keys = {}
+        self._net_spec_keys = net_spec_keys
```

Comment:
I think there should be an if statement here?

Reply:
If statement added.